Example/Authenticator/View Models/Connections/ConnectionsViewModel.swift

```diff
@@ -170,6 +175,17 @@ extension ConnectionsViewModel {
     func reconnect(id: ID) {
         delegate?.reconnect(by: id)
     }
+    
+    func checkInternetAndRemoveConnection(id: String, showConfirmation: Bool) {
+        guard reachabilityManager.isReachable else {
+            delegate?.showNoInternetConnectionAlert {
+                if self.reachabilityManager.isReachable { self.remove(by: id) }
+            }
+            return
+        }
+        if showConfirmation { delegate?.showDeleteConfirmationAlert { self.remove(by: id) } }
```

Comment:
Add new line before `if`. By our internal convention we decided to add new line after `guard` statement.

Also indent it like this:

```swift
if showConfirmation {
    delegate?.showDeleteConfirmationAlert { self.remove(by: id) }
} else {
    remove(by: id)
}
```